appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/property/MotionProperty.kt

```diff
@@ -48,16 +51,24 @@ abstract class MotionProperty<T, V : AnimationVector>(
 
     private var lastVelocity = animatable.velocity
 
+    /**
+     * Render-ready value that contains applied displacements on top of the [internalValue].
+     * Backed by compose androidx.compose.runtime.State
+     */
+    var renderValue: T by mutableStateOf(animatable.value)
```

Comment:
I had to move it up in the class due to https://github.com/bumble-tech/appyx/actions/runs/6606332442/job/17942423218#step:8:284

The flow was updating `renderValue` before it was initialised